### PR TITLE
Add graphql blogpost

### PR DESCRIPTION
**Pull request by Jotschi**
Signed-off-by: Johannes Schüth <git@jotschi.de>

### CODE REVIEW

**Comment by Jotschi** [comment]
The title is very basic but I have not yet come up with an alternative. Any suggestion and or update of the blogpost is welcome.

**Comment by Jotschi** [comment]
@vietj I have updated the title and bumped the date of the article.

File: src/site/blog/posts/2017-11-14-vertx-graphql.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+---
+title: Vert.x meets GraphQL
```
Comment (vietj) [inline src/site/blog/posts/2017-11-14-vertx-graphql.md]: _Eclipse Vert.x_ instead

Comment (Jotschi) [reply src/site/blog/posts/2017-11-14-vertx-graphql.md]: Resolved

**Pull request merged**